h_transport_materials/property_database/molybdenum.py

```diff
@@ -49,12 +49,25 @@
     source="frauenfelder_permeation_1968",
 )
 
+guthrie_p_0 = 3.8e-3 * u.ccSTP * u.cm**-1 * u.s**-1 * u.atm**-0.5
+guthrie_permeability = Permeability(
+    pre_exp=guthrie_p_0 / ((htm.Rg * 300 * u.K),
```

Comment:
Why do you need to divide by Rg * 300K?

The variable `guthrie_p_0` already has the right dimensionality for permeability